test/distributed/test_device_mesh.py

```diff
@@ -563,24 +563,33 @@ def test_get_item_3d_noncontinuous_slicing(self):
 
 
 class TestMeshEnv(DTensorTestBase):
+    @property
+    def world_size(self):
+        return 8
```

Comment:
It's better to do `return min(8, torch.cuda.device_count())`.

Reply:
Confirmed that when gpu is less than world_size, DTensorTestBase will default to "gloo" backend.